app/models/katello/glue/pulp/repo.rb

```diff
@@ -691,6 +705,22 @@ def package_lists_for_publish
          :filenames => filenames.to_set}
       end
 
+      def docker?
```

Comment:
Is there a reason that these were moved from repository.rb to repo.rb?  Is it because they are needed by a capsule which perhaps does not have a Katello::Repository ?